Translation to Mongolian Language

BigmanZaya opened the pull request:
Contribution to translate to Mongolian Language.

BigmanZaya committed:
Create widget.js

Contribution to translate to Mongolian Language

BigmanZaya commented:
Hello, if you have any question, please contact below:
email: ankhzaya@datacom.mn

dEEpProjects commented:
Hi,

Thanks for your contribution.

As described in the readme, there are different modules. In order to give to the users a better experience, could you try to take a look at the other modules too (at least **api** and **user-app**)?

We will then take care of validating the proposal and merging it to the master branch.

We are ready to help you in case of any doubts.

BigmanZaya committed:
api & user-app

As vendor wished, updated following js

BigmanZaya commented:
Thanks for the update, I have sent 2 commits. Can you check now

BigmanZaya committed:
email update

BigmanZaya committed:
Update user-welcome.html

dEEpProjects commented:
Hi,

Firstly let me congratulate you on your incredible job!

From a first check, I see that the following keys are missing the translation.

In the api module:

- **'socialLoginTitle'**
- **'emailLoginTitle'**
- **'editProfileTitle'**

In the widget module:

- **'commons.close'**
- **'status-header.editUserProfileModal'**
- **'status-header.editUserProfile'**
- **'chat.welcomeMessage'**
- **'chat.welcomeMessageJWT'**
- **'chat.userIdenticon'**

Could you check?

fvasco commented on mn/email/user-welcome.html:
```diff
@@ -0,0 +1,12 @@
+Тавтай морил!
```
Should it be "Now4real-д тавтай морил!"?

BigmanZaya replied:
Got it

fvasco commented on mn/email/user-email-login.html:
```diff
@@ -0,0 +1,14 @@
+Таны баталгаажуулах код {{#if siteName}} {{siteName}}{{/if}}
```
Extra space in "код {{#if"

BigmanZaya commented:
> In the api module:
> 
> * **'socialLoginTitle'**
> * **'emailLoginTitle'**
> * **'editProfileTitle'**

Are you sure I did not find these keys in api.js in English version?
I have updated others.

BigmanZaya committed:
Fixed somethings

dEEpProjects commented:
> > In the api module:
> > 
> > * **'socialLoginTitle'**
> > * **'emailLoginTitle'**
> > * **'editProfileTitle'**
> 
> Are you sure I did not find these keys in api.js in English version? I have updated others.

I'm sorry, I meant **user-app** module and not **api**.

BigmanZaya committed:
more update

BigmanZaya commented:
> > > In the api module:
> > > 
> > > * **'socialLoginTitle'**
> > > * **'emailLoginTitle'**
> > > * **'editProfileTitle'**
> > 
> > 
> > Are you sure I did not find these keys in api.js in English version? I have updated others.
> 
> I'm sorry, I meant **user-app** module and not **api**.

Okey, I have translated. PLease check again

dEEpProjects committed:
Refactor

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+     */
+    'monitor.pageViewersTitle': 'Хуудсыг үзэж буй хүмүүс',
+    'monitor.siteViewersTitle': 'Сайтыг үзэж буй хүмүүс',
+    'monitor.chatBalloon': 'ЧААТ', // keep upper case, if applicable
```
It's a typo? Maybe 'ЧАТ'?

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    'timeUnit.days': 'өдөр',
+    'timeUnit.weeks': 'долоо хоног',
+    'timeUnit.months': 'сар',
+    'timeUnit.years': 'жилд',
```
Maybe 'жил'?

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+
+    'timeUnit.minutes': 'минут',
+    'timeUnit.hours': 'цаг',
+    'timeUnit.days': 'өдөр',
```
Maybe 'өдрүүд'?

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    'status-header.signin': 'Орох',
+    'status-header.editUserProfile': 'Профайл засварлах',
+
+    'status-header.notRememberedLogin': 'Сануулаагүй',
```
In English we want to say that we cannot store the user and that after a page refresh the user needs to authenticate again. Is the translation right?

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    /**
+     * Tab labels and tooltips
+     */
+    'tab.chatTabLabel': 'ЧААТ', // max 8 characters; keep upper case, if applicable
```
Maybe 'ЧАТ'?

BigmanZaya replied:
Updated.

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    'tab.rankingTabLabel': 'ХУУДСУУД', // max 8 characters; keep upper case, if applicable
+    'tab.rankingTitle': 'Үзэж буй хуудсууд',
+
+    'tab.heatmapTabLabel': 'ЗУРАГ', // max 8 characters; keep upper case, if applicable
```
We refer to the world map.

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    /**
+     * Ranking
+     */
+    'ranking.foregroundTitle': 'Зочид',
```
We refer to people present on this site.

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    'ranking.foregroundTitle': 'Зочид',
+    'ranking.activeTitle': 'Чаатад буй хүмүүс',
+    'ranking.foregroundSwitchLabel': 'УНШИГЧИД', // max 12 characters; keep upper case, if applicable
+    'ranking.activeSwitchLabel': 'ЧААТЛАГЧИД', // max 12 characters; keep upper case, if applicable
```
Maybe 'чалчаа'?

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+     * Chat
+     */
+    'chat.welcomeMessage': 'Бусад зочидтой шууд чат бич. Бичсэн чат нийтэд харагдах ба $N $A хугацааны дараа алга болно.',
+    'chat.welcomeMessageJWT': 'Бусад зочидтой шууд чат бич. Бичсэн чат нийтэд харагдах ба $N $A хугацааны дараа алга болно.',
```
In this case we want to say that the chat COULD be public but it does not necessarily mean that it will be.

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    // below are the actions on messages that appear in a contextual menu
+    'chat.replyBtn': 'Хариулах', // means "reply to this message"
+    'chat.deleteBtn': 'Устгах', // means "delete this message"
+    'chat.muteBtn': 'Дууг хаах', // means "mute this user"
```
We want to say: Block the user from posting new messages

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    'presence.page.a': 'Хуудсанд 1 хүн байна',
+    'presence.page.b': 'Хуудсанд $N хүн байна',
+
+    'presence.site.a': 'Хуудсанд 1 хүн байна',
```
We refer to site and not page

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    'presence.page.b': 'Хуудсанд $N хүн байна',
+
+    'presence.site.a': 'Хуудсанд 1 хүн байна',
+    'presence.site.b': 'Хуудсанд $N хүн байна',
```
We refer to site and not page

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    /**
+     * Mute dialog
+     */
+    'mute-modal.title': 'Хэрэглэгчийг хаах',
```
We want to say: Prevent the user to writing new messages

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+
+    'mute-modal.body.deleteInfo': 'Хэрэглэгчийг хүссэн үедээ нээж болно.',
+
+    'mute-modal.body.deleteAlert': 'Та хэрэглэгчийг нээлээ',
```
We want to say: You are re-enabling the user to post new messages

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+
+    'mute-modal.headerForever': 'Хэрэглэгч $U-г хаасан',
+
+    'mute-modal.body.set': 'Хэрэглэгч $U-г хаах хугацаа:',
```
The English verb is "to mute", with the meaning of "block the user for posting new messages"

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    'mute-modal.body.update': 'Хаах хугацааг шинэчлэх:',
+    // example: "Update mute period:" followed by input fields with "5 days"
+
+    'mute-modal.body.deleteInfo': 'Хэрэглэгчийг хүссэн үедээ нээж болно.',
```
We want to say that you can re-enable the user posting at any time

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+    'mute-modal.body.set': 'Хэрэглэгч $U-г хаах хугацаа:',
+    // example: "Mute user Ben for:" followed by input fields with "3 days"
+
+    'mute-modal.body.update': 'Хаах хугацааг шинэчлэх:',
```
We mean update of the time in which the user cannot post

BigmanZaya replied:
Updated.

dEEpProjects commented on mn/widget.js:
```diff
@@ -0,0 +1,271 @@
+     */
+    'mute-modal.title': 'Хэрэглэгчийг хаах',
+
+    'mute-modal.header': 'Хэрэглэгчийн эрх $U $E хүртэл хаагдсан',
```
We refer to the time period in which the user cannot post new messages

BigmanZaya replied:
Updated.

dEEpProjects commented on mn/user-app.js:
```diff
@@ -0,0 +1,77 @@
+    'emailLoginTitle': 'Now4real Имэйл нэвтрэлт',
+    'editProfileTitle': 'Профайл засварлах - Now4real',
+
+    'genericError': 'Уучлаарай алдаа гарлаа',
```
Maybe 'Уучлаарай! Алдаа гарлаа'?

BigmanZaya replied:
Updated.

dEEpProjects commented on mn/user-app.js:
```diff
@@ -0,0 +1,77 @@
+
+    'emailSigninStep3.feedback': 'Код баталгаажсан. Сүүлийн алхам...',
+
+    'emailSigninStep3.nameInfo': 'Дэлгэцэнд харагдах нэр (нийтийн чатанд харагдах боно)',
```
Better: 'Дэлгэцэнд харагдах нэр (нийтийн чатанд харагдна)'?

BigmanZaya replied:
Updated

dEEpProjects commented on mn/user-app.js:
```diff
@@ -0,0 +1,77 @@
+
+    'editProfile.updatedFeedback': 'Таны мэдээллийг шинэчилж дууслаа. Энэ цонхыг хаана уу',
+
+    'editProfile.gravatarInfo': 'Gravatar ыг ашиглан таны зургийг харуулдаг өөрчлөхийн тулд <a>Gravatar/WordPress -т бүртгүүлнэ үү</a> .',
```
Could you underline that Gravatar is an external service?

BigmanZaya replied:
Updated.

dEEpProjects reviewed (changes requested):
I've checked all the strings (with the help of Google Translate).

I've found some phrases with a little different meaning and others that maybe can be improved.

Please, could you check?

BigmanZaya committed:
Translation update

 'timeUnit.days': 'өдөр',  // We dont say "өдрүүд" for date format. We just call it өдөр

'ranking.activeSwitchLabel': 'БУУ ХАЛАГЧИД', // It is idiom, meaning of CHATTERS

BigmanZaya committed:
Merge branch 'main' of https://github.com/BigmanZaya/localization

dEEpProjects committed:
Refactor

dEEpProjects commented:
Hi,

Thanks again for your contribution!

I've pushed a new commit with a minor refactor.

We will let you know when the language is available in the production widget.

BigmanZaya commented:
Thanks


Ankhzaya Tseden
Chief Technology Officer | Datacom Co., Ltd.,
***@***.*** | +976.95773344 |




On Tue, Feb 8, 2022 at 6:03 PM dEEpProjects ***@***.***>
wrote:

> Hi,
>
> Thanks again for your contribution!
>
> I've pushed a new commit with a minor refactor.
>
> We will let you know when the language is available in the production
> widget.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/now4real/localization/pull/1#issuecomment-1032426701>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXPDMMN57ZPZIK2O4ISJVR3U2DTA5ANCNFSM5M2J6X7A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
